packages/cli/src/databases/entities/ExecutionAnnotation.ts

```diff
@@ -0,0 +1,61 @@
+import {
+	Column,
+	Entity,
+	Index,
+	JoinColumn,
+	JoinTable,
+	ManyToMany,
+	OneToMany,
+	OneToOne,
+	PrimaryGeneratedColumn,
+	RelationId,
+} from '@n8n/typeorm';
+import { ExecutionEntity } from './ExecutionEntity';
+import type { AnnotationTagEntity } from './AnnotationTagEntity';
+import type { AnnotationTagMapping } from './AnnotationTagMapping';
+import type { AnnotationVote } from 'n8n-workflow';
+
+@Entity({ name: 'execution_annotations' })
+@Index(['execution'])
```

Comment:
Could also be next to the column for which it is, i.e. `executionId`. And have `{ unique: true }` parameter.

Reply:
For some reason it doesn't build when I add it next to executionId.
But I can move it next to `execution` relation if it's more preferred way.